x-pack/test/alerting_api_integration/common/lib/es_test_index_tool.ts

```diff
@@ -59,7 +59,10 @@ export class ESTestIndexTool {
   }
 
   async destroy() {
-    return await this.es.indices.delete({ index: this.index, ignore: [404] });
+    const indexExists = (await this.es.indices.exists({ index: this.index })).body;
```

Comment:
Is this a change in how the new es client works versus the old one? Or is this change addressing a separate issue?

Reply:
Yes, I believe the old client was more forgiving if you tried to delete an index that didn't exist. The new client throws an error if the index doesn't exist.